packages/terra-doc-template/src/DocTemplate.module.scss

```diff
@@ -21,4 +21,21 @@
     padding-right: var(--terra-doc-template-doc-card-not-empty-padding-right);
     padding-top: var(--terra-doc-template-doc-card-not-empty-padding-top);
   }
+
+  .props-table-markdown {
+    color: inherit;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+    font-size: 16px;
```

Comment:
Let's switch this to a `rem` value

Reply:
Changed! [c4aa9e9](https://github.com/cerner/terra-core/pull/2565/commits/c4aa9e9ba9e63996efa18c5ec14a16186b7c6dfd)